src/Api/Endpoints/Authenticators.cs

```diff
@@ -0,0 +1,76 @@
+using Microsoft.AspNetCore.Mvc;
+using Passwordless.Api.Authorization;
+using Passwordless.Common.Models.Authenticators;
+using Passwordless.Service;
+using Passwordless.Service.EventLog.Loggers;
+using Passwordless.Service.Features;
+using Passwordless.Service.Helpers;
+using static Microsoft.AspNetCore.Http.Results;
+
+namespace Passwordless.Api.Endpoints;
+
+public static class AuthenticatorsEndpoints
+{
+    public static void MapAuthenticatorsEndpoints(this WebApplication app)
+    {
+        var group = app.MapGroup("/authenticators")
+            .RequireCors("default")
+            .RequireSecretKey();
+
+        group.MapGet("/list", ListConfiguredAuthenticatorsAsync);
+
+        group.MapPost("/whitelist", WhitelistAuthenticatorsAsync)
+            .WithParameterValidation();
+
+        group.MapPost("/delist", DelistAuthenticatorsAsync)
+            .WithParameterValidation();
```

Comment:
issue: I'd like to find a better design for this api. I like the `/authenticators/` namespace, but I don't like whitelist and delist.

What about `/add` {isAllowed: true} and a `/remove`.

Since you technically added support for the blocklist in the db, this would just reflect that.